ui/packages/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.view.js

```diff
@@ -241,6 +237,14 @@ module.exports = Marionette.LayoutView.extend({
       'initialised',
       this.handleGoldenLayoutInitialised.bind(this)
     )
+    this.goldenLayout.on(
+      'itemCreated',
+      this.handleGoldenLayoutTabCreated.bind(this)
```

Comment:
❓ Not sure if it matters or not, but are there any problems with rebinding these functions on every `itemCreated` / `itemDestroyed` event?